app/services/sockethub-xmpp.js

```diff
@@ -190,6 +190,26 @@ export default Ember.Service.extend({
     }
   },
 
+  /**
+   * Ask for a channel's attendance list (users currently joined)
+   *
+   * @param {Space} space
+   * @param {Channel} channel
+   * @public
+   */
+  observeChannel(space, channel) {
+    let observeMsg = buildActivityObject(space, {
+      '@type': 'observe',
```

Comment:
This looks kind of weird. The term "observe" is from the IRC command, no? I don't think XMPP actually has "observe" for a channel. Also, just for normal humans "observe channel" would not suggest the action of "get attendance list" imo.

Reply:
This observes the channel for the type 'attendance' (see 3 lines below). I don't know if this is IRC specific, but it's how Sockethub is expecting it.